src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -194,6 +224,45 @@ DynamicInstrumentationMethod CaptureOptionsDialog::GetDynamicInstrumentationMeth
   ORBIT_UNREACHABLE();
 }
 
+void CaptureOptionsDialog::SetPureOrNotTracepoint(bool is_pure) {
+  ui_->tracepointCallstackCollectionCheckBox->setChecked(!is_pure);
+}
+
+bool CaptureOptionsDialog::GetPureOrNotTracepoint() const {
+  return !ui_->tracepointCallstackCollectionCheckBox->isChecked();
+}
+
+void CaptureOptionsDialog::SetTracepointCallstackMethod(
+    CaptureOptions::TracepointCallstackMethod tracepoint_callstack_method) {
+  switch (tracepoint_callstack_method) {
+    case CaptureOptions::kFramePointersTracepoint:
+      ui_->tracepointCallstackFramepointersMethodRadioButton->setChecked(true);
+      ui_->tracepointCallstackDWARFMethodRadioButton->setChecked(false);
+      break;
+    case CaptureOptions::kDWARFTracepoint:
+      ui_->tracepointCallstackDWARFMethodRadioButton->setChecked(true);
+      ui_->tracepointCallstackFramepointersMethodRadioButton->setChecked(false);
+      break;
+    default:
+      ui_->tracepointCallstackDWARFMethodRadioButton->setChecked(true);
+      ui_->tracepointCallstackFramepointersMethodRadioButton->setChecked(false);
+      break;
```

Comment:
This is the same logic as for `DWARF`. Can you combine those two cases?

```cpp
case CaptureOptions::kDWARFTracepoint:
default:
// ...
```

should work

Reply:
Done.